packages/traceloop-sdk/traceloop/sdk/tracing/tracing.py

```diff
@@ -525,35 +533,65 @@ def init_tracer_provider(resource: Resource) -> TracerProvider:
     return provider
 
 
-def init_instrumentations(should_enrich_metrics: bool):
-    init_openai_instrumentor(should_enrich_metrics)
-    init_anthropic_instrumentor(should_enrich_metrics)
-    init_cohere_instrumentor()
-    init_pinecone_instrumentor()
-    init_qdrant_instrumentor()
-    init_chroma_instrumentor()
-    init_google_generativeai_instrumentor()
-    init_haystack_instrumentor()
-    init_langchain_instrumentor()
-    init_mistralai_instrumentor()
-    init_ollama_instrumentor()
-    init_llama_index_instrumentor()
-    init_milvus_instrumentor()
-    init_transformers_instrumentor()
-    init_together_instrumentor()
-    init_redis_instrumentor()
-    init_requests_instrumentor()
-    init_urllib3_instrumentor()
-    init_pymysql_instrumentor()
-    init_bedrock_instrumentor(should_enrich_metrics)
-    init_replicate_instrumentor()
-    init_vertexai_instrumentor()
-    init_watsonx_instrumentor()
-    init_weaviate_instrumentor()
-    init_alephalpha_instrumentor()
-    init_marqo_instrumentor()
-    init_lancedb_instrumentor()
-    init_groq_instrumentor()
+def init_instrumentations(should_enrich_metrics: bool, block_instruments: Optional[Set[Instruments]] = None):
```

Comment:
ideally this might be an opportunity to combine these 2 ways and just have `init_instrumentations` work the same as the `if instruments is not None` clause - just set all instrumentations as the value of `instruments`. WDYT?